### PR TITLE
Client-side settings

### PR DESCRIPTION
**Pull request by oagna**
Linked Task: #RASPIO-48

### CODE REVIEW

File: app/app/src/main/res/menu/bottom_nav_menu.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <item
             android:id="@+id/navigation_notifications"
```
Comment (profiluefter) [inline app/app/src/main/res/menu/bottom_nav_menu.xml]:
```suggestion
            android:id="@+id/navigation_settings"
```

File: app/app/src/main/res/navigation/mobile_navigation.xml [hunk 2]
```diff
@@ -19,9 +19,8 @@
 
     <fragment
             android:id="@+id/navigation_notifications"
```
Comment (profiluefter) [inline app/app/src/main/res/navigation/mobile_navigation.xml]:
```suggestion
            android:id="@+id/navigation_settings"
```

**Review by profiluefter — CHANGES_REQUESTED** [review]
Lopm

**Review by profiluefter — APPROVED** [review]

**Review by CallMeAlasca — CHANGES_REQUESTED** [review]
Adjust Main-Activity navigation

**Review by CallMeAlasca — APPROVED** [review]

**Review by profiluefter — APPROVED** [review]

**Pull request merged**